WalletWasabi/WebClients/ChaumianCoinJoin/WasabiClient.cs

```diff
@@ -21,6 +22,29 @@ public WasabiClient(Uri baseUri, IPEndPoint torSocks5EndPoint = null) : base(bas
 		{
 		}
 
+		#region blockchain
+
+		public async Task<IEnumerable<string>> GetFiltersAsync(string bestKnownBlockHash, int count)
+		{
+			using(var response = await TorClient.SendAndRetryAsync(HttpMethod.Get,
+																	HttpStatusCode.OK, 
+																	$"/api/v1/btc/blockchain/filters?bestKnownBlockHash{bestKnownBlockHash}&count={count}"))
+			{
+				if (response.StatusCode != HttpStatusCode.OK)
```

Comment:
It produces 204 NoContent, too.

Reply:
In which case you should return IEnumerable.Empty

Reply:
What about just returning the empty list from the controller as an OK response?

Reply:
Or removing the if check that returns the the NoContent response so the empty list is passed through.

Reply:
Made this change, also returning null in case NotFound is returned.